languages/de.lang

```diff
@@ -390,6 +391,7 @@ $PALANG['pFetchmail_field_returned_text']   = 'Zurückgegebener Text';
 $PALANG['pFetchmail_desc_id']               = 'Eintrags-ID';
 $PALANG['pFetchmail_desc_mailbox']          = 'Lokales Postfach';
 $PALANG['pFetchmail_desc_src_server']       = 'Entfernter Server';
+$PALANG['pFetchmail_desc_src_service']      = 'Entfernter Service, permits you to specify a service name to connect to. You can specify a decimal port number here, if your services database lacks the required service-port assignments.';
```

Comment:
This does not match the english text - and doesn't look german ;-)

Please get the text in sync with the english text (translating it is of course ok ;-)